Gemfile

```diff
@@ -36,6 +36,8 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'devise'
```

Comment:
We usually lock the gem versions just in case an update breaks our project.
The idea is that if devise updates their gem with breaking changes, a simple bundle update could make some trouble for us, so it's better to stick to a particular version and only update it if we think it's worth it.
I usually use the version suggested in https://rubygems.org/